frontend/packages/ceph-storage-plugin/package.json

```diff
@@ -21,6 +21,9 @@
       ],
       "ceph-storage": [
         "integration-tests/**/*.scenario.ts"
+      ],
+      "e2e": [
```

Comment:
`ocp-ci-e2e` might be a better name.

Reply:
it's not something we have control over.